draft-quic-atsss-reqs.mkd

```diff
@@ -471,16 +468,58 @@ This document discusses three approaches:
 
 ### Single QUICv1 Tunnel with Unreliable Datagram Extension and Connection Migration
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+               ------------
+    *---------/            \---------*
+    |........| Access Net A |........|
+    |:*-------\   (3GPP)   /-------*:|
+    |:|        ------------        |:|
+    |v|                            |:|
+ +--+-+-+                       +--+-+-+
+ |QUIC  |                       |QUIC  |        +------+
+ |      |                       |      |.......>|Server|
+ |Client|                       |Proxy |        +------+
+ +------+                       +------+
+
+               ------------
+              /            \
+             | Access Net B |           Legend:
+              \ (non 3GPP) /               --- QUIC tunnel connection
+               ------------                ... Tunneled non-TCP flow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+{: #fig-single-quic title="Single QUICv1 tunnel"}
+
 
 In this case, there is one QUIC session between the UE and the ATSSS proxy.
 This QUIC session uses only one access network at a time. Thanks to the
 connection migration capability of QUIC, it is possible to support the
-switching and steering functions of ATSSS, but splitting cannot be supported. 
+switching and steering functions of ATSSS, but splitting cannot be supported.
 
 This is not an option for ATSSSv2.
 
 ### Multiple QUICv1 Tunnels with Unreliable Datagram Extension and Connection Migration
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+               ------------
+    *---------/            \---------*
+    |........| Access Net A |........|
+    |:*-------\   (3GPP)   /-------*:|
+    |:|        ------------        |:|
+    |v|                            |:|
+ +--+-+-+                       +--+-+-+       +------+
+ |QUIC  |                       |QUIC  |......>|      |
+ |      |                       |      |       |Server|
+ |Client|                       |Proxy |......>|      |
```

Comment:
There should be only one line between the proxy and there server as this is the same QUIC connection

Reply:
The following text states that splitting can be challenging, and it's not really whether the text consider this complexity as manageable or not. If we put a single line, we may want to revise the text. The same discussion could be held for MP-QUIC.

Reply:
Right, in this scenario it could make sense to have two e2e flows. Maybe we can reflect that in the image title like "Traffic steering of two end-to-end flows using multiple QUICv1 tunnels"?

For the MP-QUIC, however, we explicitly say that this can be used to provide smother handover by splitting, so in that image below I would recommend to display only one e2e flow.